README.md

```diff
@@ -19,6 +19,33 @@ TODO
 
 TODO
 
+## How to run tests
```

Comment:
logback.xml and application.yml use environment variables that need to be documented